dev/tests/api-functional/testsuite/Magento/GraphQl/Quote/Customer/SetShippingMethodsOnCartTest.php

```diff
@@ -7,7 +7,11 @@
 
 namespace Magento\GraphQl\Quote\Customer;
 
+use Magento\Catalog\Api\ProductRepositoryInterface;
+use Magento\Catalog\Model\Product;
+use Magento\Quote\Model\Quote\Address\Rate;
```

Comment:
`use Magento\Quote\Model\Quote\Address\Rate;` should be placed after use `Magento\Quote\Model\Quote;`